config/manifests/gateway-api-inference-extension/values.yaml

```diff
@@ -0,0 +1,13 @@
+inferenceExtension:
+  replicas: 1
+  image:
+    name: epp
+    hub: us-central1-docker.pkg.dev/k8s-staging-images/gateway-api-inference-extension
+    tag: main
+    pullPolicy: Always
+
+  name: inference-gateway-ext-proc
+
+inferencePool:
+  namespace: default
+  name: vllm-llama2-7b-pool
```

Comment:
I'm not really sure what we want our default pool name to be, but this seems to specific. Maybe `base` or `default`?

cc @ahg-g @danehans @kfswain

Reply:
`pool-1` :)